pkgs/applications/graphics/meshlab/default.nix

```diff
@@ -32,21 +40,47 @@ in stdenv.mkDerivation {
   hardeningDisable = [ "format" ];
```

Comment:
is this still needed, I'm not sure what was the motivation behind its addition

Reply:
Seems to come from https://github.com/NixOS/nixpkgs/commit/136562adab750df73f0f646fdc679bc71dcb9a68

Without it one gets:

```
../../external/qhull-2003.1/src/io.c:2169:3: error: format not a string literal and no format arguments [-Werror=format-security]
   fprintf(fp, endfmt);
   ^~~~~~~
```

I'll add a comment.

Reply:
If this is the only error maybe we should just patch it and remove the `hardeningDisable`. :)

Reply:
`endFmt` doesn't really seem to be a format so I guess we should just `fputs` it.

Reply:
I don't want to do that, because our patches will likely create conflicts with the next update, increasing maintenance burden.

You can try make an _upstream_ PR (though meshlab has lots of PRs open and they get mostly ignored while further development is done).